sfge/src/main/java/com/salesforce/apex/jorje/JorjeUtil.java

```diff
@@ -129,7 +129,7 @@ private static void visitComments(JorjeNode wrapper, CommentVisitor visitor) {
         }
     }
 
-    public static class JorjeCompilationException extends SfgeRuntimeException {
+    public static final class JorjeCompilationException extends SfgeRuntimeException {
```

Comment:
Curious: why do we want these exception classes to be final?

Reply:
In general, part of our code style agreement is that non-abstract classes should be final. Specifically here, having them be final makes it easier to preserve the desired behavior (always calling `super()` in future code changes.